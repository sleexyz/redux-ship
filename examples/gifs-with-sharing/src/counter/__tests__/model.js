// @flow
import * as Test from '../../test';
import * as CounterModel from '../model';

Test.snapshotReduce(CounterModel.reduce, {
  IncrementByOne: {
    patch: {type: 'IncrementByOne'},
    state: CounterModel.initialState,
  },
  IncrementByTwo: {
    patch: {type: 'IncrementByTwo'},
    state: CounterModel.initialState,
  },
});
