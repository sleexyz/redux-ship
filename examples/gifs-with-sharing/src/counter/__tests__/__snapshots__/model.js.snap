exports[`test IncrementByOne 1`] = `
Object {
  "count": 1
}
`;

exports[`test IncrementByTwo 1`] = `
Object {
  "count": 2
}
`;
